build/pipeline-releasebuild.yaml

```diff
@@ -46,14 +47,29 @@ steps:
     BuildPlatform: '$(BuildPlatform)'
     BuildConfiguration: '$(BuildConfiguration)'
     MsalClientSemVer: $(MsalClientSemVer)
-    Solution: 'Libs.sln'
+    Solution: 'LibsAndSamples.sln'
 
 # Run Post-build code analysis (e.g. Roslyn)
 - template: template-postbuild-code-analysis.yaml
 
 # Pack and sign all of the nuget packages
 - template: template-pack-and-sign-all-nugets.yaml
 
+- task: UseDotNet@2
+  displayName: 'Use .Net Core SDK 3.x'
+  inputs:
+    version: 3.x
+
+# Run Unit Tests
```

Comment:
Could you add a template that runs all tests? I don't think we ever run just unit tests or just integration tests. Should include cache compat tests as well if not included in integration tests already.

Reply:
I was under the impression that they are already there. Ill double check

Reply:
ill reconfigure the test template